can-define-stream_test.js

```diff
@@ -177,24 +180,22 @@ test('Test if streams are memory safe', function() {
 
 	defineStream(canStreaming)(MyMap);
 
-	var map = new MyMap();
+	var getNumberOfBindings = function(map) {
+		var meta = map[metaSymbol];
+		return meta && meta.handlers && meta.handlers.get([]).length;
+	};
 
-	QUnit.equal(map.__bindEvents._lifecycleBindings, undefined, 'Should have no bindings');
+	var map = new MyMap();
+	assert.equal(getNumberOfBindings(map), undefined, 'Should have no bindings');
 
 	var handler = function(ev, newVal, oldVal){};
-	map.on("baz", handler);
-
-	QUnit.equal(map.__bindEvents._lifecycleBindings, 3, 'Should have 3 bindings');
-
+	map.on('baz', handler);
+	assert.equal(getNumberOfBindings(map), 3, 'Should have 3 bindings');
 
 	map.off('baz', handler);
-	QUnit.equal(map.__bindEvents._lifecycleBindings, 0, 'Should reset the bindings');
```

Comment:
so @justinbmeyer I tried counting the number of handlers to replicate what was previously done with `_lifecycleBindings` but it seems that when `can-events` was used, unbinding `baz` would reset all the bindings it created.

In my case, after calling `map.on('baz', handler)` the number of handlers is two, which makes to me but brakes the previous assertion.

a) is it incorrect to replace `_lifecycleBindings` with the number or handlers? or
b) Maybe that expectation of all bindings gone when `baz` is unbound is incorrect?